File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/02.Conditional-Statements/02-exercise/05.godzillaVsKong.js
function godzillaVsKong(input) {

    const budget = Number(input[0]);
    const numberOfSupernumerary = Number(input[1]);
    const clothingPriceForOne = Number(input[2]);

    const decor = budget * 0.1;
    let clothingPrice = numberOfSupernumerary * clothingPriceForOne;

    if (numberOfSupernumerary > 150) {
        clothingPrice = clothingPrice - clothingPrice * 0.1;
    }

    const costs = decor + clothingPrice;
    const difference = Math.abs(budget - costs);

    if (costs > budget) {
        console.log("Not enough money!");
        console.log(`Wingard needs ${difference.toFixed(2)} leva more.`);
    } else {
        console.log("Action!");
        console.log(`Wingard starts filming with ${difference.toFixed(2)} leva left.`);
    }

}
godzillaVsKong(["20000", "120", "55.5"]);

File: Python_Basics/Homeworks-And-Labs/02.conditional_statements/02.exercise/05.godzilla_vs._kong.py
budget = float(input())
numberOfSupernumerary = int(input())
clothingPriceForOne = float(input())

decor = budget * 0.1
clothingPrice = numberOfSupernumerary * clothingPriceForOne

if numberOfSupernumerary > 150:
    clothingPrice = clothingPrice - clothingPrice * 0.1

costs = decor + clothingPrice
difference = abs(budget - costs)

if costs > budget:
    print("Not enough money!")
    print(f'Wingard needs {difference:.2f} leva more.')
else:
    print("Action!")
    print(f'Wingard starts filming with {difference:.2f} leva left.')